images/kubekins-e2e/Dockerfile

```diff
@@ -101,8 +101,9 @@ RUN if [ -n "${KIND_VERSION}" ]; then \
 ARG KUBETEST2_VERSION
 RUN if [ -n "${KUBETEST2_VERSION}" ]; then \
     wget -q -O /usr/local/bin/kubetest2.tgz https://storage.googleapis.com/k8s-staging-kubetest2/${KUBETEST2_VERSION}/linux-amd64.tgz && \
-    tar -xzf /usr/local/bin/kubetest2.tgz && \
-    rm -f /usr/local/bin/kubetest2.tgz; \
+    tar -xzf /usr/local/bin/kubetest2.tgz -C /usr/local/bin && \
```

Comment:
I'd you want to guarante these binaries take precedence see what kind does

Reply:
you mean, overwrite potentially existing binaries?